src/dlh_ingest.egg-info/dependency_links.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
do we need /src/dlh_ingest.egg-info dir? We should also remove all pyc files. Maybe add a gitignore for pyc files.